### ISSUE TITLE
GOME U9

### ISSUE THREAD

**Issue by Akagami95** [phhusson/treble_experimentations#1308]
External Sound does not work at all on all the phhusson based GSI for some unknown reason.....but works on stock perfectly.....from Android 9 till Android 10....it uses the Helio p23 CPU (MTK 6763).... please can you find a solution.... without the GSI.....I cannot upgrade the phone but can't use without sound

**Comment by alexmigf** [comment]
Do you have incoming SMS?

**Comment by Akagami95** [comment]
Yes I do.....and I receive and hear calls.....it's just that there is no sound coming out of the speakers...hence the phone does not ring and I can't listen to music externally

**Comment by alexmigf** [comment]
What android version?

**Comment by Akagami95** [comment]
On the android 9 and android 10 gsi

**Comment by alexmigf** [comment]
I've tried already 5/6 roms of 10 and none of them has the SMS incoming working. Let me change to 9 and see. Thank you!

**Comment by Akagami95** [comment]
Do you have the same sound issue??

**Comment by alexmigf** [comment]
On android 10 i have, indeed.

**Comment by Akagami95** [comment]
It's a very annoying issue....I hope for a fix soon

**Comment by phhusson** [comment]
Please provide logcat when starting a music on both stock and GSI
Also provide a link to stock ROM

**Comment by Akagami95** [comment]
Sorry but please how do I provide the logcat......and here is the link to the stock ROM.....thanks.....https://www.needrom.com/download/gome-u9-global-firmware-v1/

**Comment by phhusson** [comment]
Please provide a link that doesn't require registering

**Comment by Akagami95** [comment]
So.....this phone actually has two ROM versions.....the Chinese version with no support for Google apps and the international version which apparently is only available in the link above.....but the link to the Chinese ROM is http://www.mediafire.com/file/ml3p237fyrfjo4i/Gome_U9_MT6763_2018M29A_11072018_8.1.0.zip/file......hope it helps

**Comment by alexmigf** [comment]
@phhusson I will download the global and put in GDrive.

**Comment by Akagami95** [comment]
Thanks @alexmigf.....that would really help

**Comment by alexmigf** [comment]
@phhusson Here's the link for the global stock rom: https://drive.google.com/file/d/153ADydEI2OIH4FIkDLhtoEp3ap26hLFh/view?usp=sharing

**Comment by Akagami95** [comment]
Thanks in advance @phhusson.....the global ROM is the more used ROM......thank you

**Comment by phhusson** [comment]
got the img thanks

**Comment by phhusson** [comment]
Looks like it's the usual issue

Copy the following files (from stock rom) to /system/etc/smartpa_params/

**Comment by Akagami95** [comment]
Thanks @phhusson.....will test it out

**Comment by Akagami95** [comment]
Have you tried the fix for the ROM @alexmigf......and do you have incoming SMS???.....let's see how that can be resolved too

**Comment by alexmigf** [comment]
@phhusson  how do i access the system files from the img? I'm not able to mount it on mac.

**Comment by Akagami95** [comment]
@alexmigf You cant......I've tried.....you have to copy and paste manually after flashing the ROM with root explorer

**Comment by alexmigf** [comment]
@phhusson Thanks, i will test on the weekend

**Comment by Akagami95** [comment]
@alexmigf have you tested it

**Comment by alexmigf** [comment]
@Akagami95 Not yet, sorry, i will try this week or weekend.

**Comment by alexmigf** [comment]
@phhusson What files do i need to copy? Everything inside /system from the stock rom?

**Comment by phhusson** [comment]
yeah copy all files in /system/etc/smartpa_params/ from stock to gsi

**Comment by alexmigf** [comment]
@phhusson After flashing the gsi rom i got stuck on the boot android logo, in both v216 and v217. Any thoughts?

**Comment by phhusson** [comment]
get logcat

**Comment by alexmigf** [comment]
@phhusson https://we.tl/t-Tbzr5dQPpr

**Comment by phhusson** [comment]
Easy: uninstall magisk.

**Comment by alexmigf** [comment]
@phhusson No luck https://we.tl/t-ptTPoVvcsQ

**Comment by nordschle1fe** [comment]
sound in the speaker earned?  what need to do ?  what firmware to install?

**Comment by ponces** [comment]
[Fixed](https://github.com/phhusson/device_phh_treble/commit/95cdb160d546b623de0f9ae29805a52dcc51706d)  ;)
